backend/controllers/sprints.js

```diff
@@ -52,48 +47,99 @@ function validateSprint({ start_date, end_date, sprint }) {
   return null
 }
 
+const fetchSprintsFromDb = async (studentNumber) => {
+  try {
+    const latestGroupId = await findLatestGroupId(studentNumber)
+
+    const groupSprints = await db.Sprint.findAll({
+      where: { group_id: latestGroupId },
+      //add group_id to attributes if needed for frontend
+      attributes: ['id', 'start_date', 'end_date', 'sprint'],
+      raw: true
+    })
+
+    if (groupSprints.length === 0) {
+      console.error('No sprints found for the group')
+      throw new Error('No sprints found for the group')
+    }
+    const formattedSprints = groupSprints.map(sprint => ({
+      id: sprint.id,
+      start_date: new Date(sprint.start_date).toISOString().slice(0, 10),
+      end_date: new Date(sprint.end_date).toISOString().slice(0, 10),
+      sprint: sprint.sprint,
```

Comment:
What is this field? If sprint means integer 1, 2, 3 etc., consider better naming such as sprint_no

Reply:
Left as is due to risky change (w/o automated testing). Issue raised for later change.